notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,549 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GPT-Neo Text Prediction with OpenVINO\n",
+    "\n",
+    "This notebook shows a text prediction with OpenVINO. We use the [GPT-Neo](https://zenodo.org/record/5297715#.ZAmpsXZBztU) model, which is a part of the Generative Pre-trained Transformer (GPT) family. GPT-Neo is pre-trained on a large corpus of English text using unsupervised training. The model is available from [HuggingFace](https://huggingface.co/EleutherAI/gpt-neo-125M). GPT-Neo displays a broad set of capabilities, including the ability to generate conditional synthetic text samples of unprecedented quality, where we prime the model with an input and have it generate a lengthy continuation.\n",
```

Comment:
It would be good to mention that the demonstration uses three models instead of briefing just GPTNeo.

Example: This notebook demonstrates text prediction using OpenVINO with three different models: GPT-Neo, GPT-2, and PersonaGPT(update the respective links).